tests/dragonfly/snapshot_test.py

```diff
@@ -217,6 +217,32 @@ async def test_snapshot(self, df_seeder_factory, async_client, df_server):
         assert super().get_main_file("test-set-snapshot_cron-summary.dfs")
 
 
+@dfly_args({**BASIC_ARGS})
+class TestDoubleSave(SnapshotTestBase):
+    """Test set snapshot_cron flag"""
+
+    @pytest.fixture(autouse=True)
+    def setup(self, tmp_dir: Path):
+        super().setup(tmp_dir)
+
+    @pytest.mark.asyncio
+    @pytest.mark.slow
+    async def test_snapshot(self, df_seeder_factory, async_client, df_server):
+        seeder = df_seeder_factory.create(
+            port=df_server.port, keys=100000, multi_transaction_probability=0
+        )
+        await seeder.run(target_deviation=0.05)
+
+        try:
+            await asyncio.gather(
```

Comment:
please add a comment, that only one save operation at a time is alowed
also please check that one save does returns OK and the other one fails, also worth checking the error returned